plugins/autoclean.c

```diff
@@ -47,12 +47,15 @@ static struct command_result *json_autocleaninvoice(struct command *cmd,
 	cycle_seconds = *cycle;
 	expired_by = *exby;
 
+	if (cleantimer)
+		cleantimer = tal_free(cleantimer);
+
```

Comment:
Skip the if?  `cleantimer = tal_free(cleantimer);` is sufficient?

Reply:
Yes thanks, `tal_free(NULL)` is always ok.